packages/bun-usockets/src/crypto/openssl.c

```
@@ -1348,36 +1337,13 @@ SSL_CTX *us_ssl_ctx_build_raw(struct us_bun_socket_context_options_t options,
-      ssl_ctx_build_fail(ssl_context);
-      return NULL;
-    }
-    SSL_CTX_set_verify(ssl_context,
-        options.reject_unauthorized ? (SSL_VERIFY_PEER | SSL_VERIFY_FAIL_IF_NO_PEER_CERT)
-                                    : SSL_VERIFY_PEER,
-        us_verify_callback);
-
-  } else if (options.ca && options.ca_count > 0) {
+  if (options.ca && options.ca_count > 0) {
     us_ex_idx_ensure();
     SSL_CTX_set_ex_data(ssl_context, us_ctx_user_ca_ex_idx, (void *)1);
```

Cppcheck (2.21.0)

[portability] Casting non-zero decimal integer literal to pointer.

(intToPointerCast)